src/legacy/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -151,13 +153,11 @@ export function BaseMapsVisualizationProvider(serviceSettings, i18n) {
       if (!this._tmsConfigured()) {
         try {
           const tmsServices = await serviceSettings.getTMSServices();
-          const firstRoadMapLayer = tmsServices.find((s) => {
-            return s.id === 'road_map';//first road map layer
-          });
-          const fallback = firstRoadMapLayer ? firstRoadMapLayer : tmsServices[0];
-          if (fallback) {
-            this._setTmsLayer(firstRoadMapLayer);
-          }
+          const userConfiguredTmsLayer = tmsServices[0];
+          const initBasemapLayer = userConfiguredTmsLayer
```

Comment:
This will always set `tmsServices[0]` as `initBasemapLayer`. If `tmsServices[0]` does not exist then the find will iterate on an empty list. Is that the intent? Why even look for `s.id === DEFAULT_EMS_BASEMAP` then?